api.md

```diff
@@ -1007,7 +1007,20 @@ AS
         FROM conditions
         GROUP BY time_bucket('1day', timec), location, humidity, temperature;
 ```
+
+Create an up-to-date continuous aggregate view that runs whenever new data is received.
```

Comment:
I think we should add a bit more clarity around what the semantics of this negative value are (e.g., what to set it to if you want it always to be /immediately/ up-to-date, even at more extreme performance hit).

Reply:
Ah, Solar had the same comment - I updated the verbage here with an extra sentence